extensions/desktop/kde-neon-6/locale-gen

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/perl
+
+use strict;
+
+sub uniq
+{
+    my %seen;
+    grep !$seen{$_}++, @_;
+}
+
+# TODO support for KDE's in-app language switch feature.
```

Comment:
(Not a blocker, just a question) Roughly what will it take to support that feature?

Reply:
That isn't my TO-DO. I wouldn't even know where to start :(

Reply:
this was Harald IIRC, so that is an entry point